app/page.tsx

```diff
@@ -163,20 +165,18 @@ const CreateProjectModal: React.FC<{
   )
 }
 
-export default function Home() {
+const  Home = observer(() => {
```

Comment:
Note: 

To get a component/page to listen for mobx updates and be reactive, it is as simple as wrapping the Functional Component with the wrapper `observer`